extensions/protokt-extensions-lite/build.gradle.kts

```diff
@@ -42,7 +40,12 @@ protokt {
 
 kotlin {
     sourceSets {
-        val commonMain by getting {}
+        // todo: not sure why this is needed
```

Comment:
I thought I preserved the behavior where we add the core dependencies automatically - and it works for JVM code - but attempting to publish multiplatform `common` code runs into problems unless I add this manually. I think we can deal with this later.

Reply:
okay, expand the todo comment

Reply:
Figured this out.